script.js

```diff
@@ -1,9 +1,21 @@
+// list of all todo items
+
 allTasks = []; // list of all todo items
 completed = []; // list of all items that are currently marked as completed
 uncompleted = []; // list of all items that still need to be completed
 
 loadJSON(`./example.json`); // We should only allow tasks to be added when the loadJSON is finished
 
+// Event listener for day dropdown change
+document.querySelectorAll(".dayDropdown").forEach((dropdown) => {
+  dropdown.addEventListener("toggle", function () {
+    if (this.open) {
+      const selectedDay = this.id.replace("Dropdown", "");
+      renderTasks(selectedDay);
```

Comment:
TODO this shouldn't be populated on dropdown they should be rendered immediately once the data is received.